content/reference/sql/dolt-sql-procedures.md

```diff
@@ -85,6 +86,63 @@ CALL DOLT_ADD('-a');
 CALL DOLT_COMMIT('-m', 'committing all changes');
 ```
 
+## `DOLT_BRANCH()`
+
+Create a new branch from the current HEAD, or create or replace a branch from an existing branch.  
+
+Note: The `DOLT_BRANCH()` stored procedure has a subset of functionality from the [`dolt branch` command line interface](../cli#dolt-branch):
+* To list branches, use the [`DOLT_BRANCHES` system table](dolt-system-tables#dolt_branches), not the `DOLT_BRANCH()` stored procedure.
+* Renaming and deleting branches is not currently supported from the `DOLT_BRANCH()` stored procedure. Use these `dolt branch` command line interface instead.
```

Comment:
You should just fix this quick

We really don't want people modifying system tables and they're asking for how to do this